monitoring.md

```diff
@@ -0,0 +1,920 @@
+# Gravity Monitoring & Alerts
+
+## Prerequisites
+
+Docker 101, Kubernetes 101, Gravity 101.
+
+## Introduction
+
+_Note: This part of the training pertains to Gravity 5.5 and earlier._
+
+Gravity Clusters come with a fully configured and customizable monitoring and alerting systems by default. The system consists of various components, which are automatically included into a Cluster Image that is built with a single command `tele build`.
+
+
+## Overview
+
+Before getting into Gravity’s monitoring and alerts capability in more detail, let’s first discuss the various components that are involved. 
+
+There are 4 main components in the monitoring system: InfluxDB, Heapster, Grafana, and Kapacitor. 
+
+
+### InfluxDB
+
+Is an open source time series database which is used for the main data store for current and future monitoring time series data that is handled. Provides the Kubernetes service `influxdb.monitoring.svc.cluster.local.`
```

Comment:
"current and future monitoring time series data" - not sure which "current" and "future" data it refers to.

Reply:
I'll just rephrase to say `for monitoring time series data`